packages/create-webstone-app/tests/helpers/create-app-dir.ts

```diff
@@ -94,11 +94,13 @@ test("app dir exists and is empty", async () => {
     output: "",
   };
 
-  const fakeContext: Ctx = {
+  const fakeContext: Partial<Ctx> = {
     appDir: "test-app",
   };
 
-  const fakeFsExistsSync = sinon.fake.returns(true);
+  const fakeFsExistsSync = sinon.stub();
+  fakeFsExistsSync.onFirstCall().returns(true);
+  fakeFsExistsSync.onSecondCall().returns(false);
```

Comment:
That's a workaround, as the `isSveltekit` function can't be faked properly (its a named export from an esmodule). That's why I'm faking the specific calls of the stub.

That's definitely not ideal, and just a workaround